docs/learn-about-neptune/team-management.rst

```diff
@@ -1,14 +1,138 @@
-Team, project and user management
+Team, Project and User Management
 =================================
 
+This page describes the concepts of team, project and user roles and permissions as they are used in Neptune. It also provides instructions for creating projects and organizations and adding users to them.
+
+
 .. contents::
     :local:
     :depth: 1
     :backlinks: top
 
 .. _how-to_team-management_create-organization:
 
-Create an organization
+Organization Types
+------------------
+
+.. _core-concepts_organization-types:
+
+An organization is a way to centrally manage projects, users and subscriptions.
+
+Neptune has two organization types: individual and team.
+
+Individual
+^^^^^^^^^^
+* Each user is assigned individual organization with ``username`` as an organization name.
+* The user is the only member of this organization but may :ref:`invite collaborators <how-to_team-management_invite-to-project>` to projects.
+* The user can create an unlimited number of projects in the individual organization.
+
+Team
+^^^^
+* A team organization comes in handy when an entire team needs to be managed centrally.
+* Once :ref:`created <how-to_team-management_create-organization>`, a team organization can be managed by the :ref:`organization Admin <core-concepts_user-roles_organization-admin>`. This include users and subscription.
+* Only users who joined team organization can browse its content, subject to the assigned :ref:`role <core-concepts_user-roles>` in the organization or project.
+
+Learn more about :ref:`project types <core-concepts_project-types>` and :ref:`user roles <core-concepts_user-roles>`.
+
+=======
+
+User Roles
+----------
+.. _core-concepts_user-roles:
+
+There are roles in projects and in organizations.
+
+
+Roles in an organization
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Organization roles apply only to team organizations.
+
+.. _core-concepts_user-roles_organization-admin:
+
+If you have :ref:`created an organization <how-to_team-management_create-organization>`,
+you can invite people to it. Two roles are available: Admin and Member.
+
+The following table summarizes the permissions available for the organization roles.
+
+.. csv-table::
+   :header: "Permission","Admin","Member"
+   :widths: 20, 20, 20
+
+    Access level in subscription management,edit,none
```

Comment:
Those `Access levels` are unreadable, to me.
Wouldn't it be more straightforward to just say

```
Managing subscription (including invoice data),yes,no
Managing organization members,yes,no
Creating and deleting projects,yes,no
```